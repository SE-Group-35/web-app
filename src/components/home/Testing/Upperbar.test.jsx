// import React from 'react';
// import { configure, shallow } from 'enzyme';
// import Adapter from 'enzyme-adapter-react-16';
// import Upperbar from '../Upperbar';
// import { ThemeProvider } from '@material-ui/core/styles';


// configure({adapter: new Adapter()});

// describe('<Upperbar />', () => {

//     let wrapper; 
//     const theme = {} ;

//     beforeEach(() => {        
//         wrapper = shallow(<ThemeProvider theme={theme}><Upperbar/></ThemeProvider>);                
//     });

//     it('should matches the snapshot',() => {
//         expect(wrapper.getElements()).toMatchSnapshot();
//     });
          
    
    
// });