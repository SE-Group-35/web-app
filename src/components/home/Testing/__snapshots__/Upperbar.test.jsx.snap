// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Upperbar /> should matches the snapshot 1`] = `
Array [
  <ThemeProvider
    theme={Object {}}
  >
    <InnerThemeProvider>
      <Upperbar />
    </InnerThemeProvider>
  </ThemeProvider>,
]
`;
